src/riscv-integration.adoc

```diff
@@ -188,19 +188,8 @@ for software to discover and/or control the behaviour#
 [#section_uncond_integer_addr_jump]
 ==== Unconditional Integer Address Jumps
 
-The indirect jump and link <<pcc>> (<<JALR_PCC>>) instruction allows unconditional
-jumps to a target address. The target address is provided in an *x* register;
-the new address is installed in the address field of the <<pcc>>. The address of
-the instruction following the jump (*pc* + 4) is written to an *x* register.
-<<JALR_PCC>> causes an exception when a minimum sized instruction at the
-target address is not within the bounds of the <<pcc>> or the target address is
-misaligned.
-
-ifdef::cheri_v9_annotations[]
-NOTE: *CHERI v9 Note:* This instruction is now modal and shares the same
-encoding with <<JALR_CAP>> when both {cheri_base_ext_name} and
-{cheri_legacy_ext_name} are supported.
-endif::[]
+The <<JALR_MODE>> instruction allows access to standard RISC-V <<JALR>> from
+capability mode, and in legacy mode it allows access to capability mode <<JALR>>.
```

Comment:
IMHO, I think this text is a little out of place because so far in the spec we have not defined what is "mode" or "legacy", so it is a little confusing. I think we simply need to state that JALR_MODE allows jumps to an address in an *x* register.

Reply:
I think the sentence can just be shortened

Reply:
ok - change pushed